Frameworks/UIKit/UIApplicationMain.mm

```diff
@@ -320,6 +320,14 @@ void _UIApplicationShutdown() {
     [[UIApplication sharedApplication] _sendHighMemoryWarning];
 }
 
+extern "C" void UIApplicationMainHandleSuspendEvent() {
+    [[UIApplication sharedApplication] _sendSuspendEvent];
```

Comment:
Same for resume below

---
In reply to: [70830139](https://github.com/Microsoft/WinObjC/pull/612#discussion_r70830139) [](ancestors = 70830139)